setup.py

```diff
@@ -166,7 +166,11 @@ def add_mim_extension():
 
             if mode == 'symlink':
                 src_relpath = osp.relpath(src_path, osp.dirname(tar_path))
-                os.symlink(src_relpath, tar_path)
+                try:
+                    os.symlink(src_relpath, tar_path)
+                except OSError:
+                    # Symlink fails with WinError on Windows
+                    shutil.copytree(src_relpath, tar_path)
```

Comment:
It would be better to judge the `mode` by  `platform.system() == 'Windows'` in L142 instead of adding `try` here.

Reply:
solved